### PR TITLE
[M375] change radios to dropdowns and fix tests

### PR DESCRIPTION
**Pull request by tienhoah**
1) Fixed related tests, and removed irrelevant ones.
2) Replace the following radio button groups with drop downs:

**Fish belt:**
- Width
- Fish Size Bin
- Reef Slope
- Visibility
- Current
- Relative Depth
- Tide

**Bleaching:**
- Visibility
- Current
- Relative depth
- Tide

**Benthic Photo Quadrat**
- Visibility
- Current
- Relative Depth
- Tide

**Habitat Complexity**
- Reef Slope
- Visibility
- Current
- Relative Depth
- Tide

**Benthic PIT**
- Reef Slope
- Visibility
- Current
- Relative Depth
- Tide

**Benthic LIT**
- Reef Slope
- Visibility
- Current
- Relative Depth
- Tide

**Site**
- Exposure
- Reef Type
- Reef Zone

**Management Regime**
- Compliance

### CODE REVIEW

**Comment by tienhoah** [comment]
[change radio buttons to dropdowns (Awan)](https://trello.com/c/Zl8VcLR0/375-change-radio-buttons-to-dropdowns-awan)

File: src/components/pages/collectRecordFormPages/BenthicLitForm/BenthicLitTransectInputs.js [hunk 1]
```diff
@@ -37,17 +37,11 @@ const BenthicLitTransectInputs = ({
...
-    { label: 'not reported', value: '' },
-  ]
-  const relativeDepthOptions = [
-    ...getOptions(relativedepths.data),
-    { label: 'not reported', value: '' },
-  ]
-  const visibilityOptions = [...getOptions(visibilities.data), { label: 'not reported', value: '' }]
-  const currentOptions = [...getOptions(currents.data), { label: 'not reported', value: '' }]
-  const tideOptions = [...getOptions(tides.data), { label: 'not reported', value: '' }]
+  const reefSlopeSelectOptions = getOptions(reefslopes.data)
```
Comment (julia-rom) [inline src/components/pages/collectRecordFormPages/BenthicLitForm/BenthicLitTransectInputs.js:40]: nitpick on naming - `reefSlopeOptions` (or if you want to keep `Select`, add it to the others too)

File: src/components/pages/collectRecordFormPages/FishBeltForm/FishBeltTransectInputs.js [hunk 2]
```diff
@@ -58,17 +58,11 @@ const FishBeltTransectInputs = ({
...
-    { label: 'not reported', value: '' },
-  ]
-  const relativeDepthOptions = [
-    ...getOptions(relativedepths.data),
-    { label: 'not reported', value: '' },
-  ]
-  const visibilityOptions = [...getOptions(visibilities.data), { label: 'not reported', value: '' }]
-  const currentOptions = [...getOptions(currents.data), { label: 'not reported', value: '' }]
-  const tideOptions = [...getOptions(tides.data), { label: 'not reported', value: '' }]
+  const reefSlopeSelectOptions = getOptions(reefslopes.data)
```
Comment (julia-rom) [inline src/components/pages/collectRecordFormPages/FishBeltForm/FishBeltTransectInputs.js:61]: same here for `reefSlopeSelectOptions`

**Review by julia-rom — CHANGES_REQUESTED** [review]
nice job updating all these tests! the drop downs look good.

added a small comment about var names

with regards to the merge conflicts - I renamed the `BenthicAttributeTransectInputs `page to `BenthicPhotoQuadratTransectInputs`

for `FishBeltTransectInputs`, keep both changes to the imports

once that's ready let me know!

**Review by julia-rom — APPROVED** [review]
looks good! tests are also passing, nice work!

**Pull request merged**